### PR TITLE
nodeday-condenast

### PR DESCRIPTION
**Pull request by ckiss**
hi @jeffharrell, @totherik. This PR contains what we have so far for the 27th event. We might need to do some small changes to the content but, it's mostly ready to go. One question I have is related to the link to Previous NodeDay. I've created a folder(2014-intuit) containing the current version of the website, and currently the Previous NodeDay button points to that. Any thoughts? Thanks. Cristi K

### CODE REVIEW

File: 2015-condenast/index.html [hunk 1]
```diff
@@ -0,0 +1,973 @@
...
+
+                    <div class="col-sm-2 time">
+
+
+
+                    </div>
+
+                    <div class="col-sm-8">
+
+                        <h3 class="title">Travel Perkins</h3>
```
Comment (totherik) [inline 2015-condenast/index.html]: Travell Perkins has 2 "L"s , FYI.

**Comment by totherik** [comment]
Great, thanks @ckiss. WRT previous and future NodeDays, we'll probably want to enumerate them on the homepage somehow. The index page should probably be general NodeDay information along with some details (date, time, location) about the upcoming event, linking to a detail page. Once another NodeDay is announced after NYC we'll need links to both previous events, so enumerating them somehow would probably be best. Let me finish reviewing this PR and we'll see what @jeffharrell thinks, too.

**Comment by dshaw** [comment]
@totherik So `condenast.nodeday.com` or `2014-04.nodeday.com`?

**Comment by totherik** [comment]
Ahh, I wasn't even thinking subdomains. We already have the [PayPal 2014 event archived](http://nodeday.com/2014-paypal), so I was just thinking about following that pattern. But rather than archiving events _into_ a directory after the event has occurred, every new event just starts in a named directory: `{YYYY}_{host}` (which is what @ckiss did here) and the homepage is just changed to highlight whichever event is coming up. That way links are preserved, videos are archived, etc.

Are you suggesting the subdomain _in addition to_ the named directories? If so, I prefer the former `condenast.nodeday.com` or `condenast2015.nodeday.com` or the like.

**Comment by lmarkus** [comment]
`2014-04-condenast`  (Since we might have more than 1 per year)
Best of both worlds :)

**Pull request merged**